automatic/videoder/videoder.nuspec

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>videoder</id>
+    <version>1.0.9</version>
```

Comment:
Should we hardcode the current version?

Reply:
No, just add 0.0 or anything lower then the current otherwise auto updater will consider it latest.